AutoCollection/Performance.ts

```diff
@@ -46,6 +46,10 @@ class AutoCollectPerformance {
                     totalFailedRequestCount: AutoCollectPerformance._totalFailedRequestCount,
                     time: +new Date
                 };
+                if (typeof (process as any).cpuUsage === 'function'){
+                    this._lastAppCpuUsage = (process as any).cpuUsage();
```

Comment:
Do we not have types for cpuUsage?

Reply:
nope. It uses 4.2.4 in package.json, this type isn't available until 6.1.0
```
"@types/node": "4.2.4",
```

Reply:
If you'd like, we can probably upgrade this package